specs/core/1_shard-data-chains.md

```diff
@@ -205,11 +205,15 @@ def get_persistent_committee(state: BeaconState,
     later_committee = get_period_committee(state, get_shard_period_start_epoch(epoch, lookback=Epoch(1)), shard)
 
     # Take not-yet-cycled-out validators from earlier committee and already-cycled-in validators from
-    # later committee; return a sorted list of the union of the two, deduplicated
-    return sorted(set(
-        [i for i in earlier_committee if epoch % EPOCHS_PER_SHARD_PERIOD < i % EPOCHS_PER_SHARD_PERIOD]
-        + [i for i in later_committee if epoch % EPOCHS_PER_SHARD_PERIOD >= i % EPOCHS_PER_SHARD_PERIOD]
-    ))
+    # later committee; return a sorted list of the union of the two, deduplicated. If the combined size is
+    # too large, take out the validators with the earlier induction epochs (ie. delay their induction)
+    prev_subset = [i for i in earlier_committee if epoch % EPOCHS_PER_SHARD_PERIOD < i % EPOCHS_PER_SHARD_PERIOD]
+
+    validators_from_post = MAX_PERSISTENT_COMMITTEE_SIZE - len(prev_subset)
+    post_subset = [i for i in later_committee if epoch % EPOCHS_PER_SHARD_PERIOD >= i % EPOCHS_PER_SHARD_PERIOD]
+    post_subset = sorted(post_subset, key=lambda i: -(i % EPOCHS_PER_SHARD_PERIOD))[:validators_from_post]
```

Comment:
This seems backwards and might break the continuity in validating a shard requiremnt.

Reply:
I'm also a bit concerned that validators with certain values wrt `% EPOCHS_PER_SHARD_PERIOD` might be more or less likely to be trimmed from the `post_subset` and have asymmetries over time in total validation time participation.

Reply:
I think the simplest approach may well be not bothering capping the persistent committee size and letting the theoretical max size be `2 * MAX_PERSISTENT_COMMITTEE_SIZE`.

Reply:
Fair. Thoughts on the name? 
Maybe `TARGET_PERSISTENT_COMMITTEE_SIZE`?

Reply:
Sure, sounds good to me!